module/spl/spl-kmem-cache.c

```diff
@@ -986,7 +986,9 @@ spl_kmem_cache_create(char *name, size_t size, size_t align,
 		if (rc)
 			goto out;
 	} else {
+#if !defined(KMEM_CACHE_USERCOPY)
 		unsigned long slabflags = 0;
+#endif
```

Comment:
Let's pass the `slabflags` to `kmem_cache_create_usercopy()` anyway even if it's always a zero and drop this new `#if` conditional.